third_party_programs_license/third-party-programs.txt

```diff
@@ -0,0 +1,2224 @@
+Intel(r) Optimization for TensorFlow Third Party Programs File
+
+This file contains the list of third party software ("third party programs") contained in the Intel software and their required notices and/or license terms. This third party software, even if included with the distribution of the Intel software, may be governed by separate license terms, including without limitation, third party license terms, other Intel software license terms, and open source software license terms. These separate license terms govern your use of the third party programs as set forth in the "third-party-programs.txt" or other similarly-named text file.
+
+Third party programs and their corresponding required notices and/or license terms are listed below.
+
+----------------------------------------------------------------------------------
+
+1. absl_py
+
+   aws-c-common
+
+   aws-c-event-stream
+
+   aws-checksums
+
+   aws-sdk-cpp
+
+   com_github_googlecloudplatform_google_cloud_cpp
+
+   dataclasses
+
+   dlpack
+   Copyright 2017 by Contributors.
+
+   flatbuffers
+
+   gemmlowp
+
+   google-auth-library-python
+
+   google-auth-library-python-oauthlib
+
+   google-pasta
+
+   google ruy
+
+   grpc
+
+   grpcio
+
+   highwayhash
+
+   importlib-metadata
+   Copyright 2017-2019 Jason R. Coombs, Barry Warsaw
+
+   oneAPI Deep Neural Network Library (oneDNN)
+   Copyright 2016-2019 Intel Corporation.
+
+   IntelÂ® Optimization for TensorFlow*(Intel tensorflow) 
+
+   keras
+
+   nsync
+
+   portpicker
+
+   requests
+    Copyright 2019 Kenneth Reitz
+
+    rsa
+    Copyright 2011 Sybren A. Stuvel <sybren@stuvel.eu>
+
+    tensorflow-estimator
+
+    tensorboard
+
+    tensorboard-data-server
+
+    tensorboard_plugin_wit
+
+    tensorflow-io-gcs-filesystem
+
+    tensorflow
+    Copyright 2019 The TensorFlow Authors.  All rights reserved.
+
+    tensorflow/tensorflow:2.7.0 (docker base layer)
+
+    tensorflow/tensorflow:2.7.0-jupyter (docker base layer)
+
```

Comment:
@justkw,  Should above 2 lines be : 
    tensorflow/tensorflow:**2.8.0** (docker base layer)

    tensorflow/tensorflow:**2.8.0**-jupyter (docker base layer) ?

Reply:
updated